app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= provide(:title, "About") %>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= yield(:title)%> | Attinderance </title>
+  </head>
+  <body>
+    <h1>Attinderance | <%= yield(:title)%></h1>
+    <br>
+    <p>Attinderance was originally intended to be a RoR learning project with real-world application. As a college instructor, I wanted a quick and easy way to take attendance that was unobtrusive. Since my students are adults, calling out the rollcall seemed akward and offensive. Yet for scholarship and admin purposes, attendance was useful and in cases mandatory.</p>
+    <br>
+    <p>Railsbridge Boston, a workshop set up by volunteers from the Boston Rails community, inspired me to learn the framework and Ruby language. This app seemed like a good independent project. Of course, I couldn't do this myself.</p>
+    <br>
+    <p>Thanks go to: my mentor, Edward Loveall; Railsbridge oranizer, Melissa Xie, and BostonRubyWomen MeetUp </p>
+    <br>
+<h2>Site Map</h2>
+<p><%= render '/applications/side_bar' %></p>
```

Comment:
I figured out why this wasn't working. The partial needs to be in the folder called `application` (singular) and then `render 'side_bar'` will work without problems.